Apps/phtaniumrest/readme.html

```diff
@@ -51,24 +51,89 @@ <h2>Permissions for Interacting with Tanium REST API</h2>
             For a guide on how to create/manage computer groups in Tanium, <a href="https://docs.tanium.com/platform_user/platform_user/console_computer_groups.html">click here.</a></li>
           <ul>
             <li>NOTE: If the <b>group_name</b> parameter is not provided, the query will be executed on all registered IP addresses/hostnames in your Tanium instance.</li>
           </ul>
           <br>
+        <li>Parameterized Query</li>
+        <ul>
+          <li>Users can provide the parameter(s) of a Parameterized query in square brackets([parameter-1, parameter-2, ..., parameter-n]).</li>
+          <ul>
+            <li>Example: Get Process Details["parameter-1","parameter-2"] from all machines with Computer Name contains localhost</li>
+          </ul>
+          <li>Users can ignore the parameter part in the query if they want the default value to be considered. Below are the 2 ways a user can achieve this:</li>
+          <ul>
+            <li>Query: Get Process Details from all machines with Computer Name contains localhost</li>
+            <li>Query: Get Process Details["",""] from all machines with Computer Name contains localhost</li>
+          </ul>
+          <li>If a user wants to add only one parameter out of two parameters, users can keep the parameter empty. Below are the examples:</li>
+          <ul>
+            <li>Example: Get Process Details["parameter-1",""] from all machines with Computer Name contains localhost</li>
+            <li>Example: Get Process Details["","parameter-2"] from all machines with Computer Name contains localhost</li>
+          </ul>
+          <li>For two or more sensors in a query, users can select one of the below:</li>
+            <ul>
+              <li>Provide value for all the parameters of all the sensors in the query</li>
+              <ul>
+                <li>Example: Get Child Processes["parameter-1"] and Process Details["parameter-2","parameter-3"] from all machines</li>
+              </ul>
+              <li>Do not provide value for any of the parameters of any of the sensors in the query</li>
+              <ul>
+                <li>Example: Get Child Processes and Process Details from all machines</li>
+              </ul>
+              <li>Provide value for the parameters you want to provide. The parameters for which you don't want to add value, please use double quotes("")</li>
+              <ul>
+                <li>Example: Get Child Processes[""] and Process Details["SHA1", ""] from all machines</li>
+                <li>Example: Get Child Processes["csrss.exe"] and Process Details["", ""] from all machines</li>
+              </ul>
+            </ul>
+          <br>
+          <li>Scenarios:</li>
+          <ol>
+            <li>If Child Processes sensor expects 1 parameter and Process Details expects 2 parameters. But the user provides only 2 parameters instead of 3 then, action will fail with a proper error message.</li>
```

Comment:
```suggestion
            <li>If Child Processes sensor expects 1 parameter and Process Details expects 2 parameters. But the user provides only 2 parameters instead of 3, then action will fail with a proper error message.</li>
```

Reply:
The comma should go before `then`. Please fix throughout file